### PR TITLE
Feature seed db

### PR DESCRIPTION
**Pull request by AndresMWeber**
Added initial seeding to the DB:
to use:
`npm run seed`
10 users created.
100 posts created.
1000 comments created.
85 categories created.
100 games created.
500 reviews created.
2000 likes created.
75 merchants created.
1000 prices created.

### CODE REVIEW

File: server/bin/seeds.js [hunk 1]
```diff
@@ -1,43 +1,139 @@
...
+            max_players: faker.random.number({ min: 4, max: 20 }),
+            min_playtime: faker.random.number({ min: 5, max: 30 }),
+            max_playtime: faker.random.number({ min: 30, max: 180 }),
+            min_age: faker.random.number({ min: 5, max: 60 }),
+            mechanics: [faker.commerce.productMaterial()],
+            designers: [faker.commerce.department()],
+            artists: [faker.name.findName()],
+            publisher: faker.company.companyName(),
+            family: faker.commerce.productAdjective(),
+            categories: [String(databaseEntries.categories[Math.floor(databaseEntries.categories.length * Math.random())]._id)],
```
Comment (bobbypwang) [inline server/bin/seeds.js]: We can use the helper function `getRandomElement`.

File: server/bin/seeds.js [hunk 2]
```diff
@@ -1,43 +1,139 @@
...
+            content: faker.lorem.paragraph(),
+            game: getRandomElement(databaseEntries.games)._id,
+            author: getRandomElement(databaseEntries.users)._id
+        }
+    }))
+
+    await submitDocuments('likes', Like, Array.from({ length: 2000 }).map((e, i) => {
+        return {
+            game: getRandomElement(databaseEntries.games)._id,
+            user: getRandomElement(databaseEntries.users)._id
```
Comment (bobbypwang) [inline server/bin/seeds.js:107]: We need to ensure that the user has not liked the same game twice. Could we possibly add this check into the route?

**Review by bobbypwang — APPROVED** [review]

**Review by Kiwi-x-Kiwi — APPROVED** [review]
Same input as Bobby's comment from earlier.

File: server/bin/seeds.js (hunk 2 above)
Comment (AndresMWeber) [reply server/bin/seeds.js:107]: This would probably be better to do in the route I think.

**Pull request merged**